components/custom-radio-button/css/index.scss

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+@import "../../common/focus/mixin";
+@import "./mixin";
+
+.utrecht-custom-radio-button {
+  @include utrecht-custom-radio-button;
+}
+
+.utrecht-custom-radio-button:checked {
+  @include utrecht-custom-radio-button--checked;
+}
+
+.utrecht-custom-radio-button:active {
+  @include utrecht-custom-radio-button--active;
+}
+
+.utrecht-custom-radio-button:focus {
+  @include utrecht-custom-radio-button--focus;
+}
+
+.utrecht-custom-radio-button:focus-visible {
+  @include utrecht-focus-visible;
```

Comment:
I think it might make sense to create a `@mixin utrecht-custom-radio-button--focus-visible` and move the `@include utrecht-focus-visible;` there, so it becomes easier to accurately rebuild the component for different CSS selectors.